### ISSUE TITLE
invalid youtube link

### ISSUE THREAD

**Issue by derdrache** [Hexer10/youtube_explode_dart#234]
There is a type of Youtube video links that are considered invalid.

Example:
https://www.youtube.com/live/GN4g3Ur0QUU?feature=share

**Comment by Hexer10** [comment]
Do you know if this kind of url always redirects to a watch page?
For example:
`https://www.youtube.com/live/<VideoId>` -> `https://www.youtube.com/watch?v=<VideoId>`

Unfortunately I've not found this kind of urls anywhere else

**Comment by derdrache** [comment]
yes, the live url redirects to a normal watch?v= url as you specified in the example

These types of urls don't happen often because they are recorded livestreams and not many YouTubers do this.
Here you can find many examples of this url type: https://www.youtube.com/@Blocktrainer/streams

**Comment by itssidhere** [comment]
Remove the ?feature=share and it will work.

https://www.youtube.com/live/GN4g3Ur0QUU

When you share a youtube short, youtube appends this to keep track of some internal analytics.
